uobtheatre/productions/models.py

```diff
@@ -980,6 +983,14 @@ def total_tickets_sold(self) -> int:
             performance.total_tickets_sold() for performance in self.performances.all()
         )
 
+    @property
+    def shortdescription(self) -> Optional[str]:
```

Comment:
Can we do what we did previously with `excerpt_text` and `excerpt`?

The field on the model should be `short_description_text` and this property should be `short_description`. Looks nicer than all one word

Reply:
@Guidedriver0 can we just axe this completely